packages/smooth_app/lib/pages/product/edit_new_packagings.dart

```diff
@@ -192,7 +192,7 @@ class _EditNewPackagingsState extends State<EditNewPackagings>
             title: Text(appLocalizations.edit_packagings_title),
             subTitle: upToDateProduct.productName != null
```

Comment:
I wonder if there's any reason, that only product name is checked here. Should we still show brands, when product names are present? For example `unknown product, brand1, brand2`

Reply:
@WildOrangutan Please fix the issues as in #4659. We're talking about P0 issues that need to be addressed asap.
I have no specific data about how many products have no name but brands or no brands but a name, and it's not relevant anyway for the moment.
If you have strong beliefs about how we should display those specific name vs brands cases, please create a specific issue about it.

Reply:
Ok, I changed formatting to behave same as in referenced issue. Please take a look.